CyberSource/utilities/flex/utils/__init__.py

```diff
@@ -0,0 +1,4 @@
+from __future__ import absolute_import
```

Comment:
You do **NOT** need this file or this folder.

Reply:
removed the file